service_capacity_modeling/models/org/netflix/__init__.py

```diff
@@ -34,4 +35,5 @@ def models():
         "org.netflix.aurora": nflx_aurora_capacity_model,
         "org.netflix.postgres": nflx_postgres_capacity_model,
         "org.netflix.kafka": nflx_kafka_capacity_model,
+        "org.netflix.ddb": nflx_ddb_capacity_model,
```

Comment:
nit: can we spell this out to match the others (I know rds didn't but that was a miss). Perhaps `dynamodb`